packages/synapse-react-client/src/components/SchemaDrivenAnnotationEditor/template/ErrorListTemplate.tsx

```diff
@@ -0,0 +1,32 @@
+import { Alert } from '@mui/material'
+import {
+  ErrorListProps,
+  FormContextType,
+  RJSFSchema,
+  RJSFValidationError,
+  StrictRJSFSchema,
+} from '@rjsf/utils'
+import { getFriendlyPropertyName } from '../AnnotationEditorUtils'
+import React from 'react'
+
+export default function ErrorListTemplate<
```

Comment:
`ErrorListTemplate` extracted and moved without changes from SchemaDrivenAnnotationEditor for reuse